src/set_theory/game/basic.lean

```diff
@@ -495,8 +496,18 @@ inductive inv_ty (l r : Type u) : bool → Type u
 | right₁ : l → inv_ty ff → inv_ty tt
 | right₂ : r → inv_ty tt → inv_ty tt
 
+instance (l r : Type u) [is_empty l] [is_empty r] : is_empty (inv_ty l r tt) :=
+⟨λ h, by { cases h with _ _ _ _ a _ a _; exact is_empty_elim a }⟩
+
 instance (l r : Type u) : inhabited (inv_ty l r ff) := ⟨inv_ty.zero⟩
 
+@[simp] theorem default_inv_ty_eq_zero (l r : Type u) : (default : inv_ty l r ff) = inv_ty.zero :=
+rfl
```

Comment:
I feel like this, and `inv_val_is_empty`, probably shouldn't be `simp` lemmas. They are referring only to stuff we'll never see again once `has_inv` is available, right?

I guess you could `local attribute [simp]` them if it's useful to have them in the simp set until then.

Reply:
I agree that this lemma is useless. Not quite sure why I added it, if I'm being honest. It has been removed.

I disagree with `inv_val_is_empty` being useless, though. We'll have to rely heavily on `inv_ty` to prove the basic lemmas on division (what else could we rely on?), and I can see this theorem being used again.